certbot.html

```diff
@@ -41,7 +41,7 @@ <H2>Installation</h2>
 
         <p>Just run:</p>
 
-        <pre><code>apt install python-certbot-nginx</code></pre>
+        <pre><code>apt-get install python3-certbot-nginx</code></pre>
```

Comment:
Is there any particular reason for the switch from `apt` to `apt-get`?

Reply:
Not at all